src/benchmarks/micro/libraries/System.Text.RegularExpressions/Perf.Regex.Industry.cs

```diff
@@ -40,7 +40,7 @@ public static int Count(Regex r, string input)
     }
 
     /// <summary>Performance tests adapted from https://github.com/mariomka/regex-benchmark</summary>
-    [BenchmarkCategory(Categories.Libraries, Categories.NoWASM)]
+    [BenchmarkCategory(Categories.Libraries, Categories.Regex, Categories.NoWASM)]
```

Comment:
You will need the Regex category for all of the test classes, unless you don't want them included

Reply:
Oops, thanks.